picard/profile.py

```diff
@@ -31,11 +31,7 @@
 
 SettingDesc = namedtuple('SettingDesc', ('name', 'highlights'))
 
-_settings_groups = {}
-_groups_order = defaultdict(lambda: -1)
-_groups_count = 0
-_known_settings = set()
-_settings_titles = {}
+__initialized = False
```

Comment:
I may have missed it, but do we ever set this to True?

Reply:
Nope, fixed, nice catch :)